site2/docs/schema-evolution-compatibility.md

```diff
@@ -781,3 +853,168 @@ Consequently, you can upgrade the producers and consumers in **any order**.
 </tr>
 
 </table>
+
+## Schema verification for producers
```

Comment:
I've moved this part before `order of upgrading clients`, please delete lines 857-1017.